.github/workflows/selenium.yml

```diff
@@ -0,0 +1,29 @@
+name: Push latest Selenium images to quay
+
+on:
+  workflow_dispatch:
+  # Build every week on Monday 00:00
+  schedule:
```

Comment:
I'd perhaps keep it manual. we don't update that often. or maybe monthly.

Reply:
I'll change to once per month